mmseg/models/utils/embed.py

```diff
@@ -73,12 +75,17 @@ def __init__(self,
 
     def forward(self, x):
         H, W = x.shape[2], x.shape[3]
-        if H % self.patch_size[0] != 0:
-            x = F.pad(x,
-                      (0, 0, 0, self.patch_size[0] - H % self.patch_size[0]))
-        if W % self.patch_size[1] != 0:
-            x = F.pad(x,
-                      (0, self.patch_size[1] - W % self.patch_size[1], 0, 0))
+
+        # TODO: Process overlapping op
+        if not self.overlapping:
```

Comment:
Overlapping is not precise for this. 
We may consider something like `auto_pad` or `pad_to_patch_size`. 

We also need to make it an argument and add a docstring for that.